bin/i18n/metrics.rb

```diff
@@ -39,15 +39,15 @@ def self.report_filesize(file_path, sync_step)
       )
     end
 
-    # logging to CloudWatch the Completion Status of a sync step, either success or fail.
+    # logging to CloudWatch the Completion Status of a sync process, either success or fail.
     # @param status [Boolean] Whether a step has been successful or not.
-    # @param sync_step [String] Step of the sync where the method is used. Options: in, up, down, out.
+    # @param sync_step [String] Step of the sync where the method is used. Options: sync-in, sync-up, sync-down, sync-out.
     # @option sync_component [String] Specific sync component being logged.
-    # @option message [String] Exception message causing the step to fail.
-    def self.report_success(status, sync_step, sync_component = '', message = '')
+    # @option message [String] Exception message causing the process to fail.
+    def self.report_success(status, sync_step, sync_component = 'None', message = 'None')
```

Comment:
You can switch the order of the last two arguments if you don't want to specify `sync_component` when you're logging the `message` dimension.

Reply:
Made this change.